rest/v10/monetization.ts

```diff
@@ -0,0 +1,85 @@
+import type { Snowflake } from '../../globals';
+import type { APIEntitlement, APISKU } from '../../v10';
+
+/**
+ * https://discord.com/developers/docs/monetization/entitlements#list-entitlements
+ */
+export interface RESTGetAPIEntitlementsQuery {
+	/**
+	 * User ID to look up entitlements for
+	 */
+	user_id?: Snowflake | undefined;
+	/**
+	 * Optional list of SKU IDs to check entitlements for
+	 * Comma-delimited set of snowflakes
+	 */
+	sku_ids?: string | undefined;
+	/**
+	 * Retrieve entitlements before this entitlement ID
+	 */
+	before?: Snowflake | undefined;
+	/**
+	 * Retrieve entitlements after this entitlement ID
+	 */
+	after?: Snowflake | undefined;
+	/**
+	 * Number of entitlements to return (1-100)
+	 *
+	 * @default 100
+	 */
+	limit?: number | undefined;
+	/**
+	 * Guild ID to look up entitlements for
+	 */
+	guild_id?: Snowflake | undefined;
+	/**
+	 * Whether entitlements should be omitted
```

Comment:
This is likely what they meant in the docs
```suggestion
	 * Whether ended entitlements should be omitted
```